scripts/run-tests.sh

```diff
@@ -29,31 +29,38 @@ NAMESPACE="${DEPLOY_NAMESPACE}"
 REGISTRY="${GCP_REGISTRY}"
 VERSION=$(git describe --tags --always --dirty)
 GO_DIR=${GOPATH}/src/github.com/${REPO_OWNER}/${REPO_NAME}
+APP_NAME=test-app
+KUBEFLOW_VERSION=master
```

Comment:
Do you want kubeflow repo changes to potentially affect the operator tests? I'm ok with it but I usually like to have that seperation of concerns

Reply:
I see your point. I will update KUBEFLOW_VERSION to 0.2 when it is released. Does it sound ok?